venue.html.j2

```diff
@@ -15,6 +15,24 @@
     </small>
   </figure>
 
+  <h2>Hotels</h2>
+
+PGConf.dev has negotiated discounted rates with several hotels in Montreal. We recommend that you book your accommodations early, whether or not you're staying at one of the hotels below, as there are many activities around Montreal at this time of the year.
+
+  <h3>Alt Hotel Montreal</h3>
+  <p>
+  <a href="https://www.germainhotels.com/en/le-germain-hotel/montreal">The Alt Hotel</a> is located
+  at 120, rue Peel, approximately a 15 minute walk from the conference venue.  Rooms are available
```

Comment:
I think that we should format all addresses in Montreal in the English style rather than the French style since (I'm guessing) it's probably less confusing to most of our audience. So "120 Peel St" here.

Alternatively, we can consider providing both the official English address and the official French address because it may not be clear to some people what the connection is. Anyways, for reference, the English address appears to be "120 Peel St" and the French address appears to be "120, Rue Peel".